core-v1/src/main/scala/algebra/CompareWrapped.scala

```diff
@@ -0,0 +1,76 @@
+package com.rallyhealth.vapors.v1
+
+package algebra
+
+import data.{Justified, Window}
+
+import cats.Show
+import cats.data.NonEmptyList
+
+// TODO: Extract the "wrapped" and Functor capabilities from the comparison capability?
+// TODO: Since this is no longer used on the Expr class, can this be made invariant again?
+trait CompareWrapped[F[+_]] {
+
+  def extract[A](wrapped: F[A]): A
+
+  def map[A, B](fa: F[A])(ab: A => B): F[B]
+
+  def wrapConst[A](value: A): F[A]
+
+  def compare[V](
+    value: F[V],
+    window: F[Window[V]],
+  ): F[Boolean]
+}
```

Comment:
`Extract` and `WrapConst` already handle 2 of the methods in here. I should do these TODOs before merging to avoid confusion.